File: SRC/BE/food-delivery-service/src/main/java/com/trung/fooddeliveryservice/dto/response/MenuItemResponse.java
package com.trung.fooddeliveryservice.dto.response;

import lombok.*;

import java.io.Serializable;
import java.math.BigDecimal;
import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class MenuItemResponse implements Serializable {
    private Long id;
    private Long restaurantId;
    private String name;
    private String description;
    private BigDecimal price;
    private String imageUrl;
    private String category;
    private Boolean isAvailable;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: SRC/APP/app/src/main/java/com/example/omnigo/features/customer/food/data/remote/dto/MenuItemResponse.kt
package com.example.omnigo.features.customer.food.data.remote.dto

data class MenuItemResponse(
    val id: Long,
    val restaurantId: Long,
    val name: String,
    val description: String?,
    val price: Double,
    val imageUrl: String?,
    val category: String?,
    val isAvailable: Boolean,
    val createdAt: String?,
    val updatedAt: String?
)
